### ISSUE TITLE
duplicate code in binance fetchMarkets(), python: fetch_markets()

### ISSUE THREAD

**Issue by drusakov778** [ccxt/ccxt#15118]
Hello there,

- OS: OSX/ubuntu
- Programming Language version: python
- CCXT version: 1.93.102

I've noticed that entry['precision']['price'] is retrieved twice in the fetch_markets() code, in both sync and async version.
There's no bug and no functional change from this duplication, it is just a duplicate code.

this is the relevant code snapshot as in latest ccxt 1.93.102
```
           if 'PRICE_FILTER' in filtersByType:
                filter = self.safe_value(filtersByType, 'PRICE_FILTER', {})
                tickSize = self.safe_string(filter, 'tickSize')
                entry['precision']['price'] = self.precision_from_string(tickSize)
                # PRICE_FILTER reports zero values for maxPrice
                # since they updated filter types in November 2018
                # https://github.com/ccxt/ccxt/issues/4286
                # therefore limits['price']['max'] doesn't have any meaningful value except None
                entry['limits']['price'] = {
                    'min': self.safe_number(filter, 'minPrice'),
                    'max': self.safe_number(filter, 'maxPrice'),
                }
                entry['precision']['price'] = self.precision_from_string(filter['tickSize'])
```
in fetch_markets() in:
https://github.com/ccxt/ccxt/blob/master/python/ccxt/binance.py
https://github.com/ccxt/ccxt/blob/master/python/ccxt/async_support/binance.py

as you can see "entry['precision']['price']" is retrieved twice for no reason as far as I can tell.

Kind Regards,
Dmitry

**Comment by samgermain** [comment]
Thanks for bringing this to our attention. I posted a fix in https://github.com/ccxt/ccxt/pull/15149

Feel free to post a fix via a PR in the future if you notice another easy fix like this!

**Comment by drusakov778** [comment]
Thank you for quick fix. I knew that python code is autogenerated from some other code base, so I was not comfortable enough to go into another language. Great that it was amended so quickly 

> Thanks for bringing this to our attention. I posted a fix in #15149
> 
> Feel free to post a fix via a PR in the future if you notice another easy fix like this!

**Comment by samgermain** [comment]
> Thank you for quick fix. I knew that python code is autogenerated from some other code base, so I was not comfortable enough to go into another language. Great that it was amended so quickly
> 
> > Thanks for bringing this to our attention. I posted a fix in #15149
> > Feel free to post a fix via a PR in the future if you notice another easy fix like this!

It's only autogenerated for the exchange files (those at js/exchange_name.js), and in `exchange.js` on the bottom halfish of the file
